packaging/win/deluge-win64-installer.nsi

```diff
@@ -0,0 +1,215 @@
+# Deluge Windows installer script
```

Comment:
There is only two lines difference between these nsi files so there should only be one nsi script that be be called with optional argument for 32-bit build (64-bit default?)

Reply:
I am not really familiar with NSIS, I will look into that.